partner_phone_extension/views/res_partner_view.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!-- Add phone number extension fields for partner -->
+    <record id="view_partner_simple_number_extension_form" model="ir.ui.view">
+        <field name="name">view.partner.simple.number.extension.form</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_simple_form"/>
+        <field name="arch" type="xml">
+            <field name="phone" position="after">
+                <field name="phone_extension" string="Extn."/>
+            </field>
+        </field>
+    </record>
+
+    <record id="view_partner_short_form_extension_form" model="ir.ui.view">
+        <field name="name">view.partner.short form.extension.form</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_short_form"/>
+        <field name="arch" type="xml">
+            <field name="phone" position="after">
+                <field name="phone_extension"  string="Extn."/>
+            </field>
+        </field>
+    </record>
+
+    <record id="view_partner_number_extension_form" model="ir.ui.view">
+        <field name="name">view.partner.number.extension.form</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="/form/sheet/group/group/field[@name='phone']" position="after">
```

Comment:
This gives an error:

`Element '<xpath expr="/form/sheet/group/group/field[@name='phone']">' cannot be located in parent view`

see: https://github.com/odoo/odoo/blob/11.0/odoo/addons/base/res/res_partner_view.xml#L179

there are 2 groups inside the first one. Try: `/form/sheet/group/group[2]/field[@name='phone']`

Reply:
It is tricky in odoo 11 that if you install the module alone, you will not get an error. You install SMS, it spoils the view. You install sales, it spoils the view. Logically, there is no dependency on sale btw.